### PR TITLE
[Starter CLI] Add starter tracking

### PR DESCRIPTION
**Pull request by markkaylor**
### What does it do?

Adds starter url to tracking event

### CODE REVIEW

**Review by remidej — APPROVED** [review]
Make sure you also update the strapi/tracker repo with a PR like this: https://github.com/strapi/tracker/pull/12/files#diff-3811bb805a87156a0efe95a3d3ea45ccf1e87c57aa00ab485d6b1de07a157334R140

Otherwise Amplitude will not pick up the new key

**Review by alexandrebodin — APPROVED** [review]
Can you refactor a bit to pass a more generic option to the generate-new so it doesn't have the starter property everywhere

File: packages/strapi-generate-new/lib/index.js [hunk 1]
```diff
@@ -36,9 +36,12 @@ module.exports = (projectDirectory, cliArguments) => {
     strapiVersion: require('../package.json').version,
     debug: cliArguments.debug !== undefined,
     quick: cliArguments.quickstart !== undefined,
```
Comment (alexandrebodin) [inline packages/strapi-generate-new/lib/index.js:38]: you can keep template in the root scope too to keep the code to handle the template more consistent

File: packages/strapi/lib/services/metrics/sender.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ const ANALYTICS_URI = 'https://analytics.strapi.io';
  * @returns {Function} (event, payload) -> Promise{boolean}
  */
 module.exports = strapi => {
-  const { uuid, template } = strapi.config;
+  const { uuid, packageJsonStrapi } = strapi.config;
```
[the remark this answers is not in the document]
Comment (markkaylor) [reply packages/strapi/lib/services/metrics/sender.js:24]: 🤦

**Review by alexandrebodin — APPROVED** [review]
Gonna review the whole CLI at once when you are ready for it

**Pull request merged**